src/main/java/org/jenkinsci/plugins/docker/workflow/FromFingerprintStep.java

```diff
@@ -99,7 +99,7 @@ public String getToolName() {
                         }
                         if (line.startsWith("FROM ")) {
                             fromImage = line.substring(5);
-                            break;
+                            continue;
```

Comment:
Yeah - that's interesting, but would be an enhancement IMO.   I'm considering this just a fix to the error that is happening now.